src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -519,9 +519,13 @@ public void AssemblyLoad()
                 bool AssemblyLoadFlag = false;
                 AssemblyLoadEventHandler handler = (sender, args) =>
                 {
+                    Assert.Equal(AppDomain.CurrentDomain, sender);
```

Comment:
Assert.Same?

Reply:
You think I would remember from the last time you told me 😄